### PR TITLE
build: can now use local orc if installed.

### PR DESCRIPTION
**Pull request by darlentar**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -58,6 +58,17 @@ AS_IF(
 
 AC_LANG([C++])
 CXXFLAGS="$CFLAGS -std=c++17"
+LIBS="$LIBS -lorc -lsodium -lprotobuf -lsnappy -lz -llz4 -lzstd -pthread"
```
Comment (rixed) [inline configure.ac:61]: Is libsodium really required here?

File: configure.ac [hunk 2]
```diff
@@ -58,6 +58,17 @@ AS_IF(
...
+AC_LINK_IFELSE(
+  [AC_LANG_PROGRAM([#include <orc/OrcFile.hh>],
+    [[auto t = orc::Type::buildTypeFromString("test");]],
+    [[orc::WriterOptions options;]],
+    [[auto outStream = orc::writeLocalFile("test");]],
+    [[auto writer = orc::createWriter(*t, outStream.get(), options);]],
```
Comment (rixed) [inline configure.ac:67]: Are those lines really necessary to test linkability?

File: Makefile.in [hunk 3]
```diff
@@ -129,6 +129,12 @@ INSTALLED_RAMEN_PROGRAMS = \
 	src/local.x src/global.x
 INSTALLED = $(INSTALLED_BIN) $(INSTALLED_LIB) $(INSTALLED_RAMEN_PROGRAMS) META
 
+ifneq (@HAVE_ORC@,1)
+BUNDLE_LIBS = $(BUNDLE_DIR)/lib/liborc.a
```
[the remark this answers is not in the document]
Comment (darlentar) [reply Makefile.in:133]: Ok

File: configure.ac (hunk 1 above)
Comment (darlentar) [reply configure.ac:61]: No

File: configure.ac (hunk 2 above)
Comment (darlentar) [reply configure.ac:67]: Don't think so.

**Pull request merged**